### PR TITLE
Add / Update API endpoints and responses for Protocol 18

### PR DESCRIPTION
**Pull request by Shaptic**
This adds two new operation response objects:
 - liquidity pool deposit
 - liquidity pool withdraw
updates the response for change trust to reflect the new asset type, and adds the new filter on the liquidity pool list endpoint.

Reference: the [Horizon Protocol 18 API spec](https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU).

Should address stellar/go#3884.

### CODE REVIEW

File: content/api/resources/operations/object/liquidity-pool-withdraw.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Liquidity Pool Deposit Object
```
Comment (sreuland) [inline content/api/resources/operations/object/liquidity-pool-withdraw.mdx:2]: c/p, this title should be for _Withdrawal_

**Review by sreuland — APPROVED** [review]
looks good, lots of content, left one comment on minor typo in withdrawals, thanks!

File: content/api/resources/operations/object/liquidity-pool-withdraw.mdx (hunk 1 above)
Comment (Shaptic) [reply content/api/resources/operations/object/liquidity-pool-withdraw.mdx]: nice catch, thanks!

**Comment by Shaptic** [comment]
@rice2000 can you :heavy_check_mark: & merge?

**Review by rice2000 — APPROVED** [review]
Nice!

**Pull request merged**